Cargo.toml

```
@@ -31,7 +31,7 @@ compiler_builtins = { version = "0.1", optional = true }
 core = { version = "1.0", optional = true, package = "rustc-std-workspace-core" }
 
 # linux_android / linux_android_with_fallback
-[target.'cfg(all(any(target_os = "linux", target_os = "android"), not(any(target_env = "", getrandom_backend = "custom"))))'.dependencies]
+[target.'cfg(all(any(target_os = "linux", target_os = "android"), not(any(all(target_os = "linux", target_env = ""), getrandom_backend = "custom"))))'.dependencies]
```

The `target_env = ""` condition is a leftover from our attempt to support the none Linux targets, but we currently don't have it, so we probably can remove it.
```suggestion
[target.'cfg(all(any(target_os = "linux", target_os = "android"), not(getrandom_backend = "custom")))'.dependencies]
```